### PR TITLE
Add thoughtbot open source template files

### PR DESCRIPTION
**Pull request by derekprior**
- Adds LICENSE
- Adds CONTRIBUTING.md
- Adds some thoughtbot branding to README.

### CODE REVIEW

**Comment by BlakeWilliams** [comment]
LGTM!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Contributing Code
+
+Constable is a messaging application in active use at thoughtbot. We welcome
+pull requests, but ask you to **please open an issue to discuss any proposed new
+features** you would like merged upstream.
+
+To contribute code to the application:
+
+1. Fork and clone the repo.
+2. Install dependencies and setup the project by running `bin/setup`.
```
Comment (derekprior) [inline CONTRIBUTING.md]: This probably isn't applicable

**Comment by georgebrock** [comment]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (georgebrock) [inline CONTRIBUTING.md]: Very true. There isn't a `bin/setup` script.

Comment (derekprior) [inline CONTRIBUTING.md]: In fact I pulled this from constable without much editing...

**Pull request merged**